Update about page

jnewbery opened the pull request:
This PR:

- moves the introductory text and sponsors to the about page, making the front page less cluttered
- adds a 'funding' section to the about page (closes #83)

moneyball commented:
tACK. I like the improvements. Once we have the remaining todo items done I think it is ready to merge.

harding commented on _includes/associates.html:
```diff
@@ -5,7 +5,7 @@
     <ul class="associates n-column-list">
```
You can get Jekyll to sort this list for you by doing:

```diff
$ git diff
diff --git a/_includes/associates.html b/_includes/associates.html
index a603edc..419c61b 100644
--- a/_includes/associates.html
+++ b/_includes/associates.html
@@ -1,4 +1,4 @@
-{% assign associates = site.data.associates.associates %}
+{% assign associates = site.data.associates.associates | sort: "name" %}
 
 <div id="associates">
 
```

jnewbery replied:
Thanks. That's a much better idea. Note that this sorts by first name, rather than last name, but I don't have any preference either way (as long as it's a stable and arbitrary ordering 🙂)

harding replied:
I gave up on last name sorting years ago after starting to deal with the complexity of multiple cultures, but if you want that, you can either use separate fields for given and family names or you can just use an array, e.g. in the YAML:

```
- name: [Harding, David]
```

The sort code from my earlier comment works with that, but you also need to change the way the names are printed.  E.g.:

```diff
-            <h3 itemprop="name">{{ associate.name }}</h3>
+            <h3 itemprop="name">{% for part in associate.name reversed %}{{part}} {% endfor %}</h3>
```

jnewbery replied:
I think it's fine to order by first name, but thanks for sharing this!

harding commented on index.md:
```diff
@@ -8,29 +8,9 @@ layout: home
-contact us at [info@bitcoinops.org](mailto:info@bitcoinops.org).
+Bitcoin Optech works to bring the best open source technologies and techniques
+to Bitcoin-using businesses in order to lower costs and improve customer
+experiences. [Learn more](./about).
```
Nitpick: I have a slight preference for "/about" rather than "./about".  Always using absolute paths for internal links is nice because it's context-free when grepping for stuff or doing global substitutions.

jnewbery replied:
Fixed!

harding commented:
I like it also.  One suggestion (certainly not required): I think it would be better if the "Subscribe to our newsletter" content was moved to immediately above the "Member companies" sub-head.  I feel like putting it at the top looks aggressive, but putting it below a listing of some of our (hopefully) high-quality content is just providing a reasonable keep-in-touch option.

jnewbery commented:
> I think it would be better if the "Subscribe to our newsletter" content was moved to immediately above the "Member companies" sub-head.

Yes, good idea. Done!

jnewbery commented on index.md:
```diff
@@ -8,31 +8,9 @@ layout: home
-contact us at [info@bitcoinops.org](mailto:info@bitcoinops.org).
-
-{% include newsletter-signup.html %}
+Bitcoin Optech works to bring the best open source technologies and techniques
```
I think since this text appears on the about page, we could make the text even shorter and snappier on the index page. Something like:

"Bitcoin Optech helps Bitcoin users and businesses integrate scaling technologies. Learn more."

Thoughts? @harding - you raised a point about the word 'scaling' here: https://github.com/bitcoinops/bitcoinops.github.io/pull/117#issuecomment-470618659. Do you have any thoughts about its use here?

harding replied:
If you think the term is appropriate, then we shouldn't stop using it.  In any case, we can always change this page again later if we move to a different phrasing, so there's no harm in this change.

jnewbery replied:
Thanks. I've updated the text in the front page.

jnewbery commented:
Adam and Carl updated. This is ready for final review.

moneyball commented:
tACK 283cd5c

I do think that in the future, as a separate PR, we can either (a) reduce the size of the Optech logo to better match the now-much-shorter text at the top of the home page or (b) add a bit more text description back before Learn More. But, there is so much goodness in this PR, that I think we should merge now. I'll file a separate Issue for my comment above, and we can discuss it there.

jnewbery commented:
> there is so much goodness in this PR, that I think we should merge now.

We can easily merge all the changes to the associates now and wait until we're happy with the index/front page before merging those parts.

bitschmidty commented on index.md:
```diff
@@ -8,31 +8,7 @@ layout: home
-contact us at [info@bitcoinops.org](mailto:info@bitcoinops.org).
-
-{% include newsletter-signup.html %}
+Bitcoin Optech helps Bitcoin users and businesses integrate scaling technologies. [Learn more](/about).
```
My suggestion would be to add another sentence or two here. Visually this looks "empty" on the page.

jnewbery commented:
#123 is merged. This PR now just contains the changes to the front page and about page.

> Visually this looks "empty" on the page.

I agree. I wanted to make the front page short and snappy, but as it is in this PR it looks a bit empty. We could add text or increase the font size :slightly_smiling_face:

jnewbery commented:
Updated front page to @bitschmidty's suggestion here: https://github.com/bitcoinops/bitcoinops.github.io/issues/122#issuecomment-474061811 with one minor change: 'Learn more' looked a bit lonely in a paragraph on its own, so I changed it to 'Learn more about us'.

moneyball commented:
tACK. I made a trivial change.
